text/metrics/0156-counter-gauge-model.md

```diff
@@ -0,0 +1,356 @@
+# Counter, UpDownCounter, and Gauge instruments explained
+
+Counter and Gauge instruments are different in the ways they convey
+meaning, and they are interpreted in different ways.  Attributes
+applied to metric events enable further interpretation.  Because of
+their semantics, the interpretive outcome of adding an attribute for
+Counter and Gauge instruments is different.
+
+With Counter instruments, a new attribute can be introduced with
+additional measurements to subdivide a variable count.
+
+With Gauge instruments, a new attribute can be introduced with
+additional measurements to make multiple observations of a variable.
+
+The OpenTelemetry Metrics API introduces a new kind of instrument, the
+UpDownCounter, that behaves like a Counter, meaning that attributes
+subdivide the variable being counted, but their primary interpretation
+is like that of a Gauge.
+
+## Background
+
+OpenTelemetry has a founding principal that the interface (API) should
+be decoupled from the implementation (SDK), thus the Metrics project
+set out to define the meaning of metrics API events.
+
+OpenTelemetry uses the term _temporality_ to describe how Sum
+aggregations are accumulated across time, whether they are reset to
+zero with each interval (_delta_) or accumulated over a sequence of
+intervals (_cumulative_).  Both forms of temporality are considered
+important, as they offer a useful tradeoff between cost and
+reliability.  The data model specifies that a change of temporality
+does not change meaning.
+
+OpenTelemetry recognizes both synchronous and asynchronous APIs are
+useful for reporting metrics, and each has unique advantages.  When
+used with Counter and UpDownCounter instruments, there is an assumed
+relationship between the aggregation temporality and the choice of
+synchronous or asynchronous API.  Inputs to synchronous
+(UpDown)Counter instruments are the changes of a Sum aggregation
```

Comment:
+1.

I would suggest that we avoid "incremental" since it might suggest monotonicity (or we will have to put something like "incremental/decremental").
My suggestion would be "Inputs to synchronous (UpDown)Counter instruments are the deltas, while inputs to asynchronous instruments are always the absolute value (e.g. the total sum)."